### ISSUE TITLE
The color contrast ratio of the focus indicator on "Scenario" cards with the background is 1.237:1 which is less than 3:1 in Dark mode: A11y_ML.NET Model Builder_Scenario_Non Text Contrast.

### ISSUE THREAD

**Issue by AnKushSingh05** [dotnet/machinelearning-modelbuilder#3036]
### GitHubTags
#A11ySev2; #A11yTCS;#Model Builder - Machine Learning for .NET Developers;#DesktopApp; #Win32; #BM-MlnetModelBuilder-Win32-March25;#Win32;#WCAG1.4.11;#Non Text Contrast.:#A11YWCAG2.1;#A11yMAS; 

### Environment Details:
Application Name: VS Setup
Edition Windows 11 Enterprise
Version 24H2
OS build 26100.3476
Microsoft Visual Studio Enterprise 2022 (64-bit) - Current
Version 17.13.3

### Repro Steps:
**Prerequisites:**
Install latest Visual Studio 2022 with at least one of the .NET workloads (ASP.NET and web development, .NET desktop development, or .NET Core cross-platform development) and/or the ML.NET Model Builder component selected.

1. Launch visual studio.
2. Create a new .NET project or open an existing .NET project.
3. Right-click on the project in the solution explorer and Add > Machine Learning.
4. Tab till scenario.
5. Verify that the color contrast ratio of the focus indicator on "Scenario" cards with the background is 1.237:1 which is less than 3:1 in dark mode.

### Actual Result:
The color contrast ratio of the focus indicator on "Scenario" cards with the background is 1.237:1 which is less than 3:1 in dark mode.
**Note**: This issue occurs across the application whenever there are cards, links, and radio buttons present..
### Expected Result:
The color contrast ratio of the focus indicator on "Scenario" cards with the background should be greater than or equal to 3:1.
### User Impact:
When the color contrast is less than 3:1, it makes the text difficult to see. This can lead to poor visibility and hinder accessibility for users with visual impairments.

### Attachment:

![Image](https://github.com/user-attachments/assets/cde9a6a1-7b8a-472e-86a7-3390f4d07526)

![Image](https://github.com/user-attachments/assets/c9b41aa8-5534-41c9-b5cf-b43feedef032)

![Image](https://github.com/user-attachments/assets/b1711535-a841-4898-9f32-08d3305be1b8)

![Image](https://github.com/user-attachments/assets/a68dd8d8-8e18-41b3-9a9e-734cfcf5b6bd)

**Comment by AnKushSingh05** [comment]
Issue is not fixed in 
Microsoft Visual Studio Enterprise 18 (64-bit) - main
Version 18.0.0 Preview 1.0 [10613.167.main]

![Image](https://github.com/user-attachments/assets/bc050c92-e97b-4ee3-ac18-5f4ba7520fe5)

**Comment by AnKushSingh05** [comment]
Issue is still there when the keyboard focus is on the link and cards in dark mode. Here the focus border is in black color with the black color background which is failing the contrast ratio.



<img width="862" height="468" alt="Image" src="https://github.com/user-attachments/assets/d9a9a42a-ebc4-4859-949a-a5c83a7bb30d" />

<img width="984" height="461" alt="Image" src="https://github.com/user-attachments/assets/8c61d609-d7a1-4608-9f84-1fccd581fbc4" />

**Comment by AnKushSingh05** [comment]
Issue is fixed in version 17.19.2.2536801

<img width="1024" height="467" alt="Image" src="https://github.com/user-attachments/assets/27757e6e-8c21-42ca-838e-4e40e37f6f8d" />

<img width="1020" height="471" alt="Image" src="https://github.com/user-attachments/assets/0eb0fe16-6056-45e8-8927-fc9a83dfb54e" />

**Comment by AnKushSingh05** [comment]
Logged a new bug as issue is still there for radio buttons: https://github.com/dotnet/machinelearning-modelbuilder/issues/3072
